geomesa-process/geomesa-process-vector/src/main/scala/org/locationtech/geomesa/process/transform/RasterEnrichProcess.scala

```diff
@@ -0,0 +1,82 @@
+/***********************************************************************
+ * Copyright (c) 2013-2017 Commonwealth Computer Research, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, Version 2.0
+ * which accompanies this distribution and is available at
+ * http://www.opensource.org/licenses/apache2.0.php.
+ ***********************************************************************/
+
+package org.locationtech.geomesa.process.transform
+
+import java.util.Date
+
+import com.vividsolutions.jts.geom.Point
+import org.geotools.coverage.grid.GridCoverage2D
+import org.geoserver.catalog.Catalog
+import org.geotools.data.collection.ListFeatureCollection
+import org.geotools.data.simple.SimpleFeatureCollection
+import org.geotools.feature.simple.SimpleFeatureBuilder
+import org.geotools.geometry.DirectPosition2D
+import org.geotools.process.ProcessException
+import org.geotools.process.factory.{DescribeParameter, DescribeProcess, DescribeResult}
+import org.joda.time.DateTime
+import org.locationtech.geomesa.process.GeoMesaProcess
+import org.locationtech.geomesa.utils.collection.CloseableIterator
+import org.locationtech.geomesa.utils.geotools.SimpleFeatureTypes
+import org.opengis.coverage.PointOutsideCoverageException
+import org.opengis.coverage.grid.GridCoverageReader
+
+import scala.collection.JavaConversions._
+import scala.util.{Failure, Success, Try}
+
+@DescribeProcess(
+  title = "Raster Enrich Process",
+  description = "Enriches Feature Collection from Raster layer"
+)
+class RasterEnrichProcess(catalog: Catalog) extends GeoMesaProcess {
```

Comment:
yeah, we can't use geoserver code like this... one work-around is to define a type that has the method you need, like we do here: https://github.com/locationtech/geomesa/blob/master/geomesa-web/geomesa-web-core/src/main/scala/org/locationtech/geomesa/web/core/GeoMesaDataStoreServlet.scala#L25-L27
alternatively, we can move this to geomesa-geoserver